tcms/testruns/static/testruns/js/get.js

```diff
@@ -22,38 +22,28 @@ $(document).ready(() => {
     });
 
     $('.add-hyperlink-bulk').click(() => {
-        const checkboxes = selectedCheckboxes()
-        if (!checkboxes.length) {
+        const selected = selectedCheckboxes()
+        if (selected === {}) {
             return false
         }
```

Comment:
I see the No executions selected message and then this proceeds to show the Add URL modal.